### ISSUE TITLE
Unable to set the input search string on TagPicker or Basepicker 

### ISSUE THREAD

**Issue by PriyankaPuranik1** [OfficeDev/office-ui-fabric-react#4725]
### Bug Report

- __Package version(s)__: 5.54.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes
Requested priority: High
Products/sites affected: Project, CRM grid
#### Describe the issue:

The Tag Picker control does not provide a way to set the search text on render. Currently we have a control that opens the tagpicker. When user types in text we want the tag picker to open with the typed in text. However there is no api to set the text value on first render on the BasePicker. If the onInputChanged method on the Basepicker is made public then we could call it from componentDidMount of our control and set the text then. 

#### Actual behavior:

No way to set the input text value for Tag Picker or Base picker on first render

#### Expected behavior:

User should be able to set the value on first render. For this we can either expose a method setInput( like focusInput) on the TagPicker that will set the input value or the onInputChanged  can be made public so that components can call it and set focus that way.

**Comment by joschect** [comment]
Good news! This has actually been fixed with https://github.com/OfficeDev/office-ui-fabric-react/pull/4569 . Now defaultVisibleValue can be passed into inputProps and autoFill correctly supports it. One caveat is that this occurs only on first render.

**Comment by PriyankaPuranik1** [comment]
Perfect. Let me bump the fabric react version and try it out.

Thanks,
Priyanka

**Comment by PriyankaPuranik1** [comment]
I tried this out and if I set defaultVisibleValue the value appears in the picker but the suggestions that open up with no results found. Is there a way to display suggestions using the default visible value?

[cid:image001.png@01D3E2DE.2015E800]

If I delete 3 and then retype it the code hits onFilterChanged and correctly displays the suggestions but not on initial render.

Thanks,
Priyanka


From: Priyanka Puranik
Sent: Thursday, May 3, 2018 11:28 AM
To: 'OfficeDev/office-ui-fabric-react' <reply@reply.github.com>; OfficeDev/office-ui-fabric-react <office-ui-fabric-react@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [OfficeDev/office-ui-fabric-react] Unable to set the input search string on TagPicker or Basepicker (#4725)

Perfect. Let me bump the fabric react version and try it out.

Thanks,
Priyanka

**Comment by PriyankaPuranik1** [comment]
Hi,
Do I need to reactivate the earlier bug? The fix is not fixing letting the suggestions being displayed based off defaultVisibleValue value?

Thanks,
Priyanka


From: Priyanka Puranik
Sent: Thursday, May 3, 2018 12:56 PM
To: 'OfficeDev/office-ui-fabric-react' <reply@reply.github.com>; 'OfficeDev/office-ui-fabric-react' <office-ui-fabric-react@noreply.github.com>
Cc: 'Author' <author@noreply.github.com>
Subject: RE: [OfficeDev/office-ui-fabric-react] Unable to set the input search string on TagPicker or Basepicker (#4725)

I tried this out and if I set defaultVisibleValue the value appears in the picker but the suggestions that open up with no results found. Is there a way to display suggestions using the default visible value?

[cid:image001.png@01D3FE48.87DF86F0]

If I delete 3 and then retype it the code hits onFilterChanged and correctly displays the suggestions but not on initial render.

Thanks,
Priyanka


From: Priyanka Puranik
Sent: Thursday, May 3, 2018 11:28 AM
To: 'OfficeDev/office-ui-fabric-react' <reply@reply.github.com>; OfficeDev/office-ui-fabric-react <office-ui-fabric-react@noreply.github.com<mailto:office-ui-fabric-react@noreply.github.com>>
Cc: Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: RE: [OfficeDev/office-ui-fabric-react] Unable to set the input search string on TagPicker or Basepicker (#4725)

Perfect. Let me bump the fabric react version and try it out.

Thanks,
Priyanka
